File: Java/Car.java
package Java;

public class Car {
    int id;
    private int passengers;
    String license;
    Account driver;
    

    public Car(String license, Account driver){
        this.license=license;
        this.driver=driver;
    
        
    }

    public void printDataCar(){
        if( passengers != 0){
            System.out.println("License: "+license +
            "\n Driver Name: "+ driver.name +" Passangers " +passengers);
        }
    }
   
    public int getPassengers() {
        return passengers;
    }
    
    public void setPassengers(int passengers) {
        if(passengers==4){
            this.passengers = passengers;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros ");
        }
        
        
    }
}
    
 
    

File: PHP/Car.php
<?php

require_once 'Account.php';
class Car{
    public $id;
    public $license;
    public $driver;
    public $passenger;


    public function __construct($license,$driver){
        $this ->_license =$license;
        $this ->_driver = $driver;

    }
    
    public function printDataCar(){
        echo "la license:  $this ->license,driver: 
        {$this ->driver ->name},document:
        {$this ->driver->document}";
    }

}
?>


